src/test/resources/application.properties

```diff
@@ -10,9 +10,15 @@ spring.servlet.multipart.max-request-size=10MB
 file.upload-dir=${FILE_UPLOAD_DIRECTORY:./images}
 file.ddl-auto=create-drop
 configuration.mock-data=false
+configuration.email.verification=true
+application.email.username=agheventscheduler@gmail.com
+application.email.password=eogbhrohgvjwgcmf
```

Comment:
Czy to sa "produkcyjne" kredki do poczty na publicznym repozytorium? :) 

Jak tak, to zmien to zeby bralo z enva, daj nam to haslo messengerem i to moze lepiej zmien. Znajac ludzi i ich skanery, to to moze juz latac po necie :(